docker/build.mk

```diff
@@ -201,51 +201,51 @@ define fuzzer_oss_fuzz_benchmark_template
 
 .$(1)-$(2)-oss-fuzz-builder-intermediate:
 	docker build \
-    --tag $(BASE_TAG)/oss-fuzz/builders/$(1)/$(2)-intermediate \
+    --tag $(BASE_TAG)/builders/$(1)/$(2)-intermediate \
     --file=fuzzers/$(1)/builder.Dockerfile \
     --build-arg parent_image=gcr.io/fuzzbench/oss-fuzz/$($(2)-project-name)@sha256:$($(2)-oss-fuzz-builder-hash) \
-    $(call cache_from,${BASE_TAG}/oss-fuzz/builders/$(1)/$(2)-intermediate) \
+    $(call cache_from,${BASE_TAG}/builders/$(1)/$(2)-intermediate) \
     fuzzers/$(1)
 
 .pull-$(1)-$(2)-oss-fuzz-builder-intermediate:
 	docker pull $(BASE_TAG)/oss-fuzz/builders/$(1)/$(2)-intermediate
```

Comment:
How about this,
search more of "oss-fuzz/builders", oss-fuzz/runners ?
git grep on entire codebase.

Reply:
I did. sorry stale commit.